frontends/election-manager/src/api.ts

```diff
@@ -19,3 +25,64 @@ export function useApiClient(): ApiClient {
   }
   return apiClient;
 }
+
+export const getAuthStatus = {
+  queryKey(): QueryKey {
+    return ['getAuthStatus'];
+  },
+  useQuery() {
+    const apiClient = useApiClient();
+    return useQuery(this.queryKey(), () => apiClient.getAuthStatus(), {
+      refetchInterval: 100,
+    });
+  },
+} as const;
+
+export const checkPin = {
+  useMutation() {
+    const apiClient = useApiClient();
+    const queryClient = useQueryClient();
+    return useMutation(apiClient.checkPin, {
+      async onSuccess() {
+        // Not strictly needed right now
```

Comment:
Maybe elaborate a bit more on why this isn't needed? Our future selves might be puzzled

Reply:
Oh oops! I dropped this comment here during our sync earlier and forgot to come back and clean it up 🤦‍♂️